addons/account/models/account_journal_dashboard.py

```diff
@@ -542,8 +542,8 @@ def _fill_sale_purchase_dashboard_data(self, dashboard_data):
             return
         bills_field_list = [
             "account_move.journal_id",
-            "(CASE WHEN account_move.move_type IN ('out_refund', 'in_refund') THEN -1 ELSE 1 END) * account_move.amount_residual AS amount_total",
-            "(CASE WHEN account_move.move_type IN ('in_invoice', 'in_refund', 'in_receipt') THEN -1 ELSE 1 END) * account_move.amount_residual_signed AS amount_total_company",
+            "(CASE WHEN account_move.move_type IN ('in_invoice', 'in_refund', 'in_receipt') THEN -1 ELSE 1 END) * account_move.amount_residual_signed AS amount_total",
+            "(CASE WHEN account_move.move_type IN ('out_refund', 'in_refund') THEN -1 ELSE 1 END) * account_move.amount_residual AS amount_total_company",
```

Comment:
I looked at it quickly since you mentioned it on the other PR.
To me it looks like you just rename the 2 columns (`amount_total` -> `amount_total_company` and vice versa). Since `amount_residual_signed` is in company currency I don't think the new names would be correct (and the runbot is red).
I did not really understand your issue from the PR description. Could you please give an example where it gives the wrong result?